moto/route53domains/validators.py

```diff
@@ -0,0 +1,1044 @@
+    "finance",
+    "financial",
+    "fish",
+    "fitness",
+    "flights",
+    "florist",
+    "flowers",
+    "fm",
+    "football",
+    "forsale",
+    "foundation",
+    "fund",
+    "furniture",
+    "futbol",
+    "fyi",
+    "gallery",
+    "games",
+    "gift",
+    "gifts",
+    "gives",
+    "glass",
+    "global",
+    "gmbh",
+    "gold",
+    "golf",
+    "graphics",
+    "gratis",
+    "green",
+    "gripe",
+    "group",
+    "guide",
+    "guitars",
+    "guru",
+    "haus",
+    "healthcare",
+    "help",
+    "hiv",
+    "hockey",
+    "holdings",
+    "holiday",
+    "host",
+    "hosting",
+    "house",
+    "im",
+    "immo",
+    "immobilien",
+    "industries",
+    "info",
+    "ink",
+    "institute",
+    "insure",
+    "international",
+    "investments",
+    "io",
+    "irish",
+    "jewelry",
+    "juegos",
+    "kaufen",
+    "kim",
+    "kitchen",
+    "kiwi",
+    "land",
+    "lease",
+    "legal",
+    "lgbt",
+    "life",
+    "lighting",
+    "limited",
+    "limo",
+    "link",
+    "live",
+    "loan",
+    "loans",
+    "lol",
+    "maison",
+    "management",
+    "marketing",
+    "mba",
+    "media",
+    "memorial",
+    "mobi",
+    "moda",
+    "money",
+    "mortgage",
+    "movie",
+    "name",
+    "net",
+    "network",
+    "news",
+    "ninja",
+    "onl",
+    "online",
+    "org",
+    "partners",
+    "parts",
+    "photo",
+    "photography",
+    "photos",
+    "pics",
+    "pictures",
+    "pink",
+    "pizza",
+    "place",
+    "plumbing",
+    "plus",
+    "poker",
+    "porn",
+    "press",
+    "pro",
+    "productions",
+    "properties",
+    "property",
+    "pub",
+    "qpon",
+    "recipes",
+    "red",
+    "reise",
+    "reisen",
+    "rentals",
+    "repair",
+    "report",
+    "republican",
+    "restaurant",
+    "reviews",
+    "rip",
+    "rocks",
+    "run",
+    "sale",
+    "sarl",
+    "school",
+    "schule",
+    "services",
+    "sex",
+    "sexy",
+    "shiksha",
+    "shoes",
+    "show",
+    "singles",
+    "site",
+    "soccer",
+    "social",
+    "solar",
+    "solutions",
+    "space",
+    "store",
+    "studio",
+    "style",
+    "sucks",
+    "supplies",
+    "supply",
+    "support",
+    "surgery",
+    "systems",
+    "tattoo",
+    "tax",
+    "taxi",
+    "team",
+    "tech",
+    "technology",
+    "tennis",
+    "theater",
+    "tienda",
+    "tips",
+    "tires",
+    "today",
+    "tools",
+    "tours",
+    "town",
+    "toys",
+    "trade",
+    "training",
+    "tv",
+    "university",
+    "uno",
+    "vacations",
+    "vegas",
+    "ventures",
+    "vg",
+    "viajes",
+    "video",
+    "villas",
+    "vision",
+    "voyage",
+    "watch",
+    "website",
+    "wedding",
+    "wiki",
+    "wine",
+    "works",
+    "world",
+    "wtf",
+    "xyz",
+    "zone",
+)
+
+VALID_DOMAIN_REGEX = re.compile(
+    r"(?:[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9]"
+)
+PHONE_NUMBER_REGEX = re.compile(r"\+\d*\.\d{10}$")
```

Comment:
```suggestion
PHONE_NUMBER_REGEX = re.compile(r"\+\d+\.\d+$")
```
Minor niggle - but phone numbers are notoriously hard to validate, that's why it stood out to  me.

The exact length of a phone number differs by country, so I think we should be very liberal in what we accept